internal/weldr/api_test.go

```diff
@@ -26,16 +29,48 @@ var packages = rpmmd.PackageList{
 	{Name: "package2"},
 }
 
-func testRoute(t *testing.T, api *weldr.API, method, path, body string, expectedStatus int, expectedJSON string) {
+func externalRequest(method, path, body string) *http.Response {
+	client := http.Client{
+		Transport: &http.Transport{
+			DialContext: func(_ context.Context, _, _ string) (net.Conn, error) {
+				return net.Dial("unix", "/run/weldr/api.socket")
+			},
+		},
+	}
+
+	req, _ := http.NewRequest(method, "http://localhost"+path, bytes.NewReader([]byte(body)))
```

Comment:
When running the test without composer running, I get a panic somewhere further down. It's fine to use `panic` in test code, but please do so here with a descriptive message (whatever the error from this method is as string is fine). Same a couple of lines down.

Reply:
Okay. To make sure I understand properly, you're saying I shouldn't ignore possible errors when creating the request and when sending it. Instead, I can use `panic(error)` to quit the execution and print out the error. This should be fixed now.

Reply:
Indeed. Error is now perfectly understandable:
```
panic: Get http://localhost/api/status: dial unix /run/weldr/api.socket: connect: permission denied [recovered]
	panic: Get http://localhost/api/status: dial unix /run/weldr/api.socket: connect: permission denied
```